cvat/apps/engine/serializers.py

```diff
@@ -1119,12 +1122,13 @@ class TaskWriteSerializer(WriteOnceMixin, serializers.ModelSerializer):
     project_id = serializers.IntegerField(required=False, allow_null=True)
     target_storage = StorageSerializer(required=False, allow_null=True)
     source_storage = StorageSerializer(required=False, allow_null=True)
+    consensus_jobs_per_segment = serializers.IntegerField(required=False, allow_null=True)
```

Comment:
Could you rename this parameter to something without "segment" in the name? Segments exist only in the server implementation, they are not exposed anywhere in the API. I can suggest `consensus_jobs_per_normal_job`.

Reply:
Sure, I will rename it to `consensus_jobs_per_normal_job`